purchases/src/main/kotlin/com/revenuecat/purchases/common/verification/SigningManager.kt

```diff
@@ -75,7 +81,33 @@ internal class SigningManager(
     fun createRandomNonce(): String {
         val bytes = ByteArray(NONCE_BYTES_SIZE)
         SecureRandom().nextBytes(bytes)
-        return String(Base64.encode(bytes, Base64.DEFAULT))
+        return String(Base64.encode(bytes, Base64.DEFAULT)).trim()
```

Comment:
This `trim` was needed for the backend integration tests that were added to work... Basically we were always creating nonces with a final `\n` character. It worked on Android fine it seems since the HttpUrlConnection implementation works different than the one in unit tests. But it was complaining in the tests.

I've confirmed that this doesn't break anything in our test app and with our integration tests.

Reply:
It might help to write a test for this?